pkg/dispatchserver/config.go

```diff
@@ -36,8 +36,11 @@ type serverConfig struct {
 	DisableHTTP       bool   `mapstructure:"disable-http" json:"disable-http"`
 	TLSPort           int    `mapstructure:"tls-port" json:"tls-port"`
 	EnableTLS         bool   `mapstructure:"enable-tls" json:"enable-tls"`
+	LetsEncrypt       bool   `mapstructure:"lets-encrypt" json:"lets-encrypt"`
+	Production        bool   `mapstructure:"production" json:"production"`
```

Comment:
IMHO you can drop this flag. staging environment is used for testing, e.g. when you are developing automation for that. you could have an internal constant, or a config flag to set the let's encrypt server, but production server should be the default.